app/api/endpoints.py

```diff
@@ -149,7 +149,8 @@ def type(
     offset: int = 0,
     limit: int = Query(10, le=18),
 ):
+    slugified_type = string_helper.slugify(type, "-")
     slugified_sort = string_helper.slugify(sort, "-")
```

Comment:
Yea, one last change
convert to lowercase?

```py
slugified_type = string_helper.slugify(type.lower(), "-")
slugified_sort = string_helper.slugify(sort.lower(), "-")
```

Reply:
But I think `slugify()` do it or did I make a mistake?
```
    def slugify(self, string: str, symbol: str) -> str:
        """Makes lowercase and strip spaces and replace space between words with "+" symbol"""
        string = string.lower().strip()
        string = re.sub(r"[^\w\s-]", "", string)
        string = re.sub(r"[\s_-]+", f"{symbol}", string)
        string = re.sub(r"^-+|-+$", "", string)
        return string
```